packages/framer-motion-3d/src/render/utils/scrape-motion-value.ts

```diff
@@ -17,14 +17,15 @@ const valueMap = {
 }
 
 export const scrapeMotionValuesFromProps: ScrapeMotionValuesFromProps = (
-    props: ThreeMotionProps
+    props: ThreeMotionProps,
+    prevProps: ThreeMotionProps
 ) => {
     const motionValues = {}
 
     for (const key in props) {
         const prop = props[key]
 
-        if (isMotionValue(prop)) {
+        if (isMotionValue(prop) || isMotionValue(prevProps[key])) {
             motionValues[valueMap[key] || key] = prop
         } else if (Array.isArray(prop)) {
```

Comment:
Do we need to handle the case where an array of motion values switches to an array of staic values?